README.md

```diff
@@ -151,7 +151,8 @@ alias precmd 'set prompt="`powerline-shell --shell tcsh $?`"'
 ### Config File
 
 Powerline-shell is customizable through the use of a config file. This file is
-expected to be located at `~/.powerline-shell.json`. You can generate the
+expected to be located at `~/.powerline-shell.json` or 
+`$XDG_CONFIG_HOME/powerline-shell/config.json`. You can generate the
```

Comment:
I'd say that it would be better to delete the `~/.powerline-shell.json` reference in order to promote the new location and escape from "the tyranny of the default".